src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -1,44 +1,62 @@
 {% from "./caseStatus.njk" import caseStatusForClaimant, caseStatusForDefendant %}
-{% from "timeRemaining.njk" import timeRemaining %}
+
+{% from "./claimDetails/noReponse.njk" import noResponseForClaimant %}
+{% from "./claimDetails/moreTimeRequested.njk" import moreTimeRequestedForClaimant %}
+
+{% from "./claimDetails/ccjEligible.njk" import ccjEligibleForClaimant %}
+{% from "./claimDetails/ccjRequested.njk" import ccjRequestedForClaimant, ccjRequesterForDefendant %}
+
+{% from "./claimDetails/claimRejected.njk" import
+claimRejectedAlreadyPaidForClaimant,
```

Comment:
Question to self: Can a `*` be used in place of this

Reply:
Clean Code - Page 307
```
J1: Avoid Long Import Lists by Using Wildcards
If you use two or more classes from a package, then import the whole package with
import package.*;
Long lists of imports are daunting to the reader. We don’t want to clutter up the tops of our modules with 80 lines of imports. Rather we want the imports to be a concise statement about which packages we collaborate with.

```

Reply:
odd normally * imports in java are discouraged, they obscure whats being imported and additional things you didn't want are now in scope.

IDEs generally collapse imports by default so readers don't need to read them